README.md

```diff
@@ -217,6 +217,79 @@ If `combineTextAndOptionsResults` was true, the first value for `runner.currentR
 
 If a `handleCommand` function was supplied, it would be called and we would skip straight from the OptionsResult to the last TextResult.
 
+### Minimal full parser
+
+This short script uses YarnBound to implement an interactive command
+line user interface for Yarn files.
+
+```
+YarnBound = require('yarn-bound');
+
+const yarnData = `
+title: start
+---
+Hello stranger, what brings you into town?
+-> What do you do here?
+    <<jump do>>
+-> Thanks, bye.
+    <<jump bye>>
+
+===
+title: do
+---
+Why, I'm the town merchant, of course. Anything you need to buy or sell, you just come talk to me.
+-> Got it.
+  <<jump start>>
+
+===
+title: bye
+---
+Goodbye then!
+<<stop>>
+===
+`;
+
+const runner = new YarnBound({
+  dialogue: yarnData,
+  startAt: 'start',
+})
+
+let options = false;
+
+process.stdin.on('data', (e) => {
```

Comment:
I think this becomes hard to understand once we get so far into the weeds regarding node processes and whatnot (this isn't just a node tool, after all, and front-end people won't have any idea what this code means). I myself am not sure why we need to `parseInt(e.toString().substring(0, 1))`. I think this illustrates why I chose to limit the scope of the documentation as I did; individual needs will be different and I don't think this (or any) specific implementation is general enough to be a helpful addition to the docs. If there is a need for a command-line runner like you've constructed, I would rather see it as a separate package.